src/Clapi/Types.hs

```diff
@@ -35,6 +45,75 @@ instance MonadFail (Either String) where
 type Attributee = String
 type Site = String
 
+class UMsg a where
```

Comment:
I can't remember what the `U` is for in this name :confused:

Reply:
I think it was for unused, the intention being to swap everything over and drop the 'u's.